### ISSUE TITLE
Mails received by coordinators of projects

### ISSUE THREAD

**Issue by Holofractographic** [FreedomCoop/valuenetwork#247]
Hi there is a issue related to the mails that OCP send to the coordinators of a project, they are receiving the mails that is send to the new participants registered throught the forms created for that. So there is any way to avoid to receive these mails? (only the one referred to the credentials when a new user is created)

**Comment by bhaugen** [comment]
Yes, on your profile settings, you can turn off any notification emails you don't want(I think this part is now only in the admin app, not the work app).
https://ocp.freedomcoop.eu/notification/settings/

If those settings are too broad (turn off too many emails), we can create a new setting that is more specific.
